parameters.toml

```diff
@@ -0,0 +1,110 @@
+# General protocol parameters.
+[parameters]
+native_token = "NAAN"
+# Minimum number of blocks in an epoch.
+# This should be 10 hours given that blocks are on average 10 seconds. ( (10*60*60)/10 )
+min_num_of_blocks = 3_600
+# Maximum expected time per block (in seconds).
+max_expected_time_per_block = 10
+# Max payload size, in bytes, for a tx.
+max_tx_bytes = 1048576
+# Max payload size, in bytes, for a tx batch proposal.
+max_proposal_bytes = 6291456
+# vp allowlist
+vp_allowlist = []
+# tx allowlist
+tx_allowlist = []
+# Implicit VP WASM name
+implicit_vp = "vp_implicit"
+# Expected number of epochs per year (also sets the min duration of an epoch in seconds)
+# Epochs should be 12 hours. ( (365*24*60*60)/(12*60*60) )
+epochs_per_year = 730
+# Maximum number of signature per transaction
+max_signatures_per_transaction = 15
+# Max gas for block
+max_block_gas = 20000000
+# Fee unshielding gas limit
+fee_unshielding_gas_limit = 20000
+# Fee unshielding descriptions limit
+fee_unshielding_descriptions_limit = 15
+
+# Map of the cost per gas unit for every token allowed for fee payment
+[parameters.minimum_gas_price]
+nam = "0.000001"
+
+# Proof of stake parameters.
+[pos_params]
+# Maximum number of active validators.
+max_validator_slots = 257
+# Pipeline length (in epochs). Any change in the validator set made in
+# epoch 'n' will become active in epoch 'n + pipeline_len'.
+pipeline_len = 2
+# Unbonding length (in epochs). Validators may have their stake slashed
+# for a fault in epoch 'n' up through epoch 'n + unbonding_len'.
+# 3 days for unbonding
```

Comment:
Nits: I would use "misbehavior" instead of "fault", and also technically the actual unbonding period for stake is unbonding_len + cubic_slashing_window_length, so 3 is incorrect here.

Reply:
the files are copied from `genesis/localnet` - if we want to change the comments we should do it there

Reply:
This line of "3 days for unbonding" isn't even in the `genesis/localnet/parameters.toml`. Perhaps this was copied from an older version? Anyway I still think we should remove it here incase others are reading this file.

Reply:
fixed